File: gtktest.js
var gobject = Packages.org.gnome.gir.gobject.GObjectGlobals.init()
var gtk = Packages.org.gnome.gir.dynamic.Gtk;

gtk.GtkGlobals.init(null, null)

var win = gtk.Window();

var button = gtk.Button.newWithLabel("hello!");
button.connect(gtk.Button.Clicked({
	    onClicked: function () {
		print("Hello");
	    }}));

win.add(button);
win.showAll();

win.connect(gtk.Widget.DeleteEvent({
	    onDeleteEvent: function (event) {
		gtk.GtkGlobals.mainQuit();
		return true;
	    }}));

gtk.GtkGlobals.main();

File: gtktest.py
from org.gnome.gir.dynamic.WebKit import *
from org.gnome.gir.dynamic.Gtk import *
from org.gnome.gir.gobject import *

GtkGlobals.initCheck(None, None)
win = Window(WindowType.TOPLEVEL)
#class DeleteHandler(Widget.DeleteEvent):
#  def onDeleteEvent(self, w, e):
#    GtkGlobals.mainQuit()
#win.connect(DeleteHandler())
sw = ScrolledWindow(None, None)
win.add(sw)
wv = WebView()
wv.open("http://www.gnome.org")
sw.add(wv)
win.setSizeRequest(640, 480)
win.showAll();
GtkGlobals.main();
